README.md

```diff
@@ -34,38 +34,52 @@ Set Hotel information using `<pf-hotel-meta>` tag:
     hotel-id="h123"
     name="Example Hotel"
     star-rating="4.5"
-    city="Neddick"
-    country="US">
+    partner-rating="5.1"
+    address-city="Neddick"
+    address-line="123 Main Street"
+    address-region="Neddick West"
+    address-country="US">
 </pf-hotel-meta>
 ```
 
 Display Room Price Freeze Buttons tag `<pf-room-button>`:
 ```html
 <pf-room-button
+    room-id="room-123"
     name="Priced Room"
     merchant-id="m890"
     beds="1"
     price="200.00"
     availability="9"
     image-url="/a/i/room/8.jpg">
-</pf-room-button><
+</pf-room-button>
 ```
 
+### Debugging:
+
+Non-production versions of the SDK will log validation warnings when invalid attributes are applied to the above elements.
+
 ### Event Listening:
 ```js
 document.addEventListener('pf-init', function() {
-  console.warn("Hopper SDK Initialized");
+  console.info("Hopper SDK Initialized");
 });
 
 document.addEventListener('pf-offer', function(evt) {
-  console.warn("Price Freeze Offer", evt.detail);
+  console.info("Price Freeze Offer", evt.detail);
 });
 
 document.addEventListener('pf-opened', function(evt) {
-  console.warn("Opened Price Freeze", evt.detail);
+  console.info("Opened Price Freeze", evt.detail);
 });
 
 document.addEventListener('pf-closed', function(evt) {
-  console.warn("Closed Price Freeze", evt.detail);
+  console.info("Closed Price Freeze", evt.detail);
+});
+
+document.addEventListener('pf-button-click', function(evt) {
```

Comment:
Whats the difference between this and the `pf-opened` event?

Reply:
functionally very little. i added this because i understood you'd discussed being able to detect click events with them. shall i remove?